homeassistant/components/ovo_energy/__init__.py

```diff
@@ -35,7 +35,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     try:
         authenticated = await client.authenticate(
-            entry.data[CONF_USERNAME], entry.data[CONF_PASSWORD]
+            entry.data[CONF_USERNAME],
+            entry.data[CONF_PASSWORD],
+            entry.data[CONF_ACCOUNT],
```

Comment:
Previous config entries wouldn't have this property right? Wouldn't this fail with a key error?

Reply:
Good point. That's adjusted